src/Paket.Core/PaketConfigFiles/InstallModel.fs

```diff
@@ -58,20 +70,21 @@ module ReferenceOrLibraryFolder =
       { old with ReferenceOrLibraryFolder.FrameworkReferences = Set.add item old.FrameworkReferences }
 
 /// Represents a subfolder of a nuget package that provides files (content, references, etc) for one or more Target Profiles.  This is a logical representation of the 'net45' folder in a NuGet package, for example.
-type FrameworkFolder<'T> =
-    { Path : ParsedPlatformPath
-      Targets : TargetProfile list
-      FolderContents : 'T  }
-
+type FrameworkFolder<'T> = { 
+    Path : ParsedPlatformPath
+    Targets : TargetProfile list
+    FolderContents : 'T
+} with
```

Comment:
Don was wrong, it doesn't work without `with`
![](http://i.imgur.com/QQj98CT.png)
I wish that it did